.npmrc

```diff
@@ -0,0 +1,4 @@
+# Requires authentication with the GitHub NPM package registry
+#
+# https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages
+@schnz:registry=https://npm.pkg.github.com
```

Comment:
Can these not be published to npm, i've not come across any other open source project that is consuming js/ts dependencies from outside the standard NPM registry.

Reply:
Already tried yesterday to move to the public npm registery. But its 40 packages (with transitive dependencies) and I got 429 rate-limited after publishing the first 25 packages. Rate limit continued to be active for more then 10 hours. Just _now_ I was able to publish the remaining packages. Will be changed in the next commit.